lib/generators/rolemodel/soft_destroyable/templates/soft_destroyable_behavior.rb

```diff
@@ -41,6 +41,20 @@
     end
   end
 
+  describe '#assign_attributes' do
+    it 'ignores other attributes when _soft_destroy is present' do
```

Comment:
Why?
I suppose if one is doing an `_destroy`, they're unlikely to pass in any other attributes (and they'd likely be irrelevant since the record would theoretically be deleted). Is that always the desired thing for `_soft_destroy` though?